SNIFFER/sniffer.py

```diff
@@ -0,0 +1,41 @@
+import scapy.all as scapy
+from scapy.layers import http  # pip install scapy_http
+
+# from scapy_http import http
+
+def get_url(packet):
+    return packet[http.HTTPRequest].Host + packet[http.HTTPRequest].Path         #for detecting urls
```

Comment:
Comment should be above the code, not beside it.

Reply:
sir, I have made changes, do I have to make any other  changes?